libs/coin-modules/coin-solana/src/signOperation.ts

```diff
@@ -72,6 +72,16 @@ export const buildSignOperation =
         });
 
         const signedTx = signOnChainTransaction(signature);
+        console.log(
```

Comment:
Let's not forget to remove it.

Reply:
Sure i will remove all dev console logs later.